tmva/pymva/test/CMakeLists.txt

```diff
@@ -128,6 +129,29 @@ endif()
 		COMMAND ./emitFromKeras
 		USES_TERMINAL
 	)
+
+   # Testing SOFIE Custom Operator functionality
+   add_executable(emitCustomModel
+                  EmitCustomModel.cxx
+                  ${CMAKE_SOURCE_DIR}/tmva/pymva/src/RModelParser_Keras.cxx
+                  ${CMAKE_SOURCE_DIR}/tmva/sofie/src/RModel.cxx
+                  ${CMAKE_SOURCE_DIR}/tmva/sofie/src/SOFIE_common.cxx
+   )
+   target_link_libraries(emitCustomModel ${PYTHON_LIBRARIES} ${Libraries})
+   target_include_directories(emitCustomModel PRIVATE
```

Comment:
You need to add these lines (as in the test before) for APPLE :
```
if(APPLE)
   target_link_options(emitCustomModel PRIVATE ${PYTHON_LINK_OPTIONS_Development_Main})
endif()
```

Reply:
made the change, thanks!